<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QDialog" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
 border: 1px solid gray;
border-radius: 3px;
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 3px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="ConfimpushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>209</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 166, 116);color: white;</string>
    </property>
    <property name="text">
     <string>Підтвердить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>78</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>Ім'я</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginlineEdit">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>108</y>
      <width>46</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginPasswordlineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gotosignuppushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Зареєструватись</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>Логін</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
